readthedocs/builds/models.py

```diff
@@ -258,32 +256,25 @@ def get_subdomain_url(self):
     def get_downloads(self, pretty=False):
         project = self.project
         data = {}
-        if pretty:
-            if project.has_pdf(self.slug):
-                data['PDF'] = project.get_production_media_url('pdf', self.slug)
-            if project.has_htmlzip(self.slug):
-                data['HTML'] = project.get_production_media_url(
-                    'htmlzip',
-                    self.slug,
-                )
-            if project.has_epub(self.slug):
-                data['Epub'] = project.get_production_media_url(
-                    'epub',
-                    self.slug,
-                )
-        else:
-            if project.has_pdf(self.slug):
-                data['pdf'] = project.get_production_media_url('pdf', self.slug)
-            if project.has_htmlzip(self.slug):
-                data['htmlzip'] = project.get_production_media_url(
-                    'htmlzip',
-                    self.slug,
-                )
-            if project.has_epub(self.slug):
-                data['epub'] = project.get_production_media_url(
-                    'epub',
-                    self.slug,
-                )
+
+        def prettify(k):
+            return k if pretty else k.lower()
+
+        if project.has_pdf(self.slug):
+            data[prettify('PDF')] = project.get_production_media_url(
+                'pdf',
+                self.slug,
+            )
+        if project.has_htmlzip(self.slug):
+            data[prettify('HTML')] = project.get_production_media_url(
```

Comment:
```suggestion
            data[prettify('HTMLZIP')] = project.get_production_media_url(
```

I believe this should be `HTMLZIP` instead of `HTML`.  The [template that displays these links checks for `dict.htmlzip`](https://github.com/rtfd/readthedocs.org/blob/ee5ffd0d5e9951da95942fa1badb796ac3f58484/readthedocs/templates/core/project_downloads.html#L12).

Reply:
Yeah, I just realized that when prettify was set, we named this to html instead of htmlzip. I need to see if we use this in the footer api, or maybe it's safe to rename the template.

Reply:
We currently use the name `HTML` in the footer api and on the theme https://github.com/rtfd/sphinx_rtd_theme/blob/a49a812c8821123091166fae1897d702cdc2d627/sphinx_rtd_theme/versions.html#L17-L21

So, it's safe to rename this in the template

Reply:
https://github.com/rtfd/readthedocs.org/pull/5579